src/web/components/App.tsx

```diff
@@ -472,6 +487,29 @@ export const App = ({ CAPI, NAV }: Props) => {
 					</>
 				</HydrateOnce>
 			))}
+
+			{inStickyNavBackscroll && (
```

Comment:
Key parts here - depending on which variant (if any), we hydrate the relevant component.